charts/stable/minio/values.yaml

```diff
@@ -29,21 +20,19 @@ probes:
   readiness:
     path: "/minio/health/ready"
```

Comment:
TLDR: this needs to reference the correct port.
Not sure you you can use tpl though...